vagrant.yml

```diff
@@ -12,6 +12,8 @@ synced_folders:
     # If using the 'smb2' type above the user and share will be configured automatically.
     smb_username: 'vagrant'
     smb_password: 'vagrant'
+    # Create smb share on vagrant up. Set to false if you already have smb shared folder.
+    smb_create: true
```

Comment:
Please name this "smb2_auto", since it's related strictly to smb2, not smb in general.